engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java

```diff
@@ -580,6 +580,7 @@ public ApplicationId launchApp(AppFactory appConfig) throws Exception
     loadDependencies();
     Configuration conf = propertiesBuilder.conf;
     conf.setEnum(StreamingApplication.ENVIRONMENT, StreamingApplication.Environment.CLUSTER);
+    propertiesBuilder.loadPlugins();
```

Comment:
This misses other ways in which application can be launched such as local mode or launch from api.